release.yaml

```diff
@@ -1,23 +1,23 @@
 fleet:
-- 100.0.4+up0.3.10-rc2
+- 100.0.4+up0.3.10-rc3
 - 100.0.0+up0.3.6
 - 100.0.1+up0.3.7
 - 100.0.2+up0.3.8
 - 100.0.3+up0.3.9
 - 0.3.900+up0.3.9
 - 0.3.900+up0.3.9
-- 100.0.4+up0.3.10-rc2
+- 100.0.4+up0.3.10-rc3
 fleet-agent:
-- 100.0.4+up0.3.10-rc2
+- 100.0.4+up0.3.10-rc3
 - 100.0.0+up0.3.6
 - 100.0.1+up0.3.7
 - 100.0.2+up0.3.8
 - 100.0.3+up0.3.9
 - 0.3.900+up0.3.9
 - 0.3.900+up0.3.9
-- 100.0.4+up0.3.10-rc2
+- 100.0.4+up0.3.10-rc3
```

Comment:
It does not look like breaking anything, but I wonder why  `100.0.4+up0.3.10-rc3` appears twice under the fleet and fleet-agent list? Can you try removing the duplication?

Reply:
Changed